### ISSUE TITLE
two selects in a row don't save correct value

### ISSUE THREAD

**Issue by rv-zjohnson** [AlecAivazis/survey#36]
If you have two select blocks in a row and on the second selection you choose the first option without moving your cursor at all, it will save the value from the first selection block instead.

```
var simpleQs = []*survey.Question{
	{
		Name: "color",
		Prompt: &survey.Select{
			Message: "Choose a color:",
			Options: []string{"red", "blue", "green"},
		},
		Validate: survey.Required,
	},
	{
		Name: "color2",
		Prompt: &survey.Select{
			Message: "Choose a color:",
			Options: []string{"red", "blue", "green"},
		},
		Validate: survey.Required,
	},
}

func main() {
	answers := struct {
		Color  string
		Color2 string
	}{}
	// ask the question
	err := survey.Ask(simpleQs, &answers)

	if err != nil {
		fmt.Println(err.Error())
		return
	}
	// print the answers
	fmt.Printf("%s and %s.\n", answers.Color, answers.Color2)
}
```

In that example, if you select "blue" and then "red" the expected output is "blue and red" and the actual output is "blue and blue".

**Comment by AlecAivazis** [comment]
Hey @rv-zjohnson thanks for reporting this. I was able to reproduce it thanks to your example and am working on a fix now. I'll let you know as soon as I have more informatino

**Comment by AlecAivazis** [comment]
@rv-zjohnson I think I have fixed the problem in #37 - please let me know if you run into any more problems

**Comment by rv-zjohnson** [comment]
Thanks @AlecAivazis!
